app/views/shared/_common_edition_attributes.html.erb

```diff
@@ -1,22 +1,33 @@
 <%= hidden_field_tag "return_to", params[:return_to] if params[:return_to] %>
+
 <div class="form-group">
   <label for="edition_assigned_to_id">Assigned to</label>
   <%= f.select :assigned_to_id, enabled_users_select_options, {}, {:class => 'form-control input-md-3', :disabled => @resource.locked_for_edits?, "data-module" => 'assignee-select'} %>
 </div>
+
 <%= render partial: 'reviewer_field', locals: { f: f } if @resource.in_review? %>
 <%= render partial: 'major_change_fields', locals: { f: f } if @resource.published_edition %>
 
 <%= f.input :title,
-      :input_html => { :disabled => @resource.locked_for_edits?, class: 'input-md-7' } %>
+  :input_html => { :disabled => @resource.locked_for_edits?, :class => 'input-md-7' } %>
 
-<%= f.input :in_beta,
-      as: :boolean,
-      label: 'Content is in beta',
-      input_html: { disabled: @resource.locked_for_edits? },
-      wrapper_html: { class: "emphasised-field form-group add-bottom-margin input-md-7" } %>
+<div class="form-group">
+  <%= f.label :phase %>
+  <%= f.select :phase, Edition::PHASE_NAMES, {},
+    :class => "form-control input-md-3",
+    :disabled => @resource.locked_for_edits? %>
+</div>
```

Comment:
I wonder if we may need to explain what a phase is to users? It feels like a not very explanatory term. I imagine it'd be good to have hint text as per meta description, maybe we can get our content designer to help?

Reply:
Since we weren't giving any hint before, it seems OK continue like that. I do think "Development" might throw people. Perhaps "Content phase"?